manim/mobject/geometry.py

```diff
@@ -1462,9 +1532,14 @@ def construct(self):
     """
 
     def __init__(self, *vertices, color=BLUE, **kwargs):
-        VMobject.__init__(self, color=color, **kwargs)
-        # There are actually four corners, and the first one is repeated twice to form the four vertices.
-        self.set_points_as_corners([*vertices, vertices[0]])
+        self.vertices = vertices
+        super().__init__(color=color, **kwargs)
+
+    def init_points(self):
+        verts = self.vertices
+        self.set_points_as_corners([*verts, verts[0]])
```

Comment:
Did this *have* to be changed? Because I would prefer setting the corners in `__init__` instead, particularly because there could be confusion with `mob.vertices` and `mob.get_vertices()`, though that falls into a larger complaint about manim's architecture I have

Reply:
Didn't have to be changed, updated in 75950e95bb994581687ffa147e6906376640d1fa :-)